ui_framework/components/local_nav/_local_calendar.scss

```diff
@@ -0,0 +1,145 @@
+.kuiDatePicker {
+  background-color: transparent;
+  border-collapse: collapse;
+  border-spacing: 0;
+  line-height: $globalLineHeight;
+}
+
+.kuiDatePickerNavigationCell {
+  padding: 0;
+}
+
+.kuiDatePickerNavigation {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: $localCalendarBottomSpacing;
+}
+
+/**
+ * 1. Override inherited styles.
+ */
+.kuiDatePickerNavigationButton {
+  appearance: none;
+  background-color: transparent;
+  border: none;
+  font-size: $globalSubTextFontSize;
+  color: $localNavButtonTextColor;
+  padding: $localCalendarButtonPadding;
+  border-radius: $globalBorderRadius;
+
+  &:hover,
+  &:active {
+    cursor: pointer;
+    color: $localNavButtonTextColor-isHover;
+    background-color: $localCalendarButtonHoverBackgroundColor;;
+  }
+
+  &:focus {
+    @include focus($globalFocusColor, $localDropdownBackgroundColor);
+
+    color: $localNavButtonTextColor; /* 1 */
+  }
+
+  @include darkTheme {
+    color: $localNavButtonTextColor--darkTheme;
+
+    &:hover,
+    &:active {
+      color: $localNavButtonTextColor-isHover--darkTheme;
+      background-color: $localCalendarButtonHoverBackgroundColor--darkTheme;
+    }
+
+    &:focus {
+      @include focus($globalFocusColor, $localDropdownBackgroundColor--darkTheme);
+
+      color: $localNavButtonTextColor--darkTheme; /* 1 */
+    }
+  }
+}
+
+.kuiDatePickerHeaderCell {
+  padding: 0;
+  color: $localNavTextColor;
+  font-size: $globalSubTextFontSize;
+  font-weight: bold;
+  text-align: center;
+
+  @include darkTheme {
+    color: $globalTextColor--darkTheme;
+  }
+}
+
+.kuiDatePickerRowCell {
+  padding: 0;
+  text-align: center;
+
+  &.kuiDatePickerRowCell-isPrecluded {
+    background-color: $localCalendarCellPrecludedBackgroundColor;
+    pointer-events: none;
+
+    @include darkTheme {
+      background-color: $localCalendarCellPrecludedBackgroundColor--darkTheme;
+    }
+  }
+}
+
+/**
+ * 1. Override inherited styles.
+ */
+.kuiDatePickerRowCellContent {
+  appearance: none;
+  background-color: transparent;
+  width: 100%;
+  border: 1px solid transparent;
+  color: $localNavTextColor;
+  font-size: $globalSubTextFontSize;
+  padding: $localCalendarCellPadding;
+  border-radius: $globalBorderRadius;
+  line-height: $localCalendarCellLineHeight;
+
+  &:focus {
+    @include focus($globalFocusColor, $localDropdownBackgroundColor);
+
+    color: $localNavTextColor; /* 1 */
+  }
+
+  &:disabled {
+    pointer-events: none;
+    opacity: 0.5;
+  }
+
+  &.kuiDatePickerRowCellContent-isOtherMonth {
+    visibility: hidden;
+    pointer-events: none;
+  }
+
+  &.kuiDatePickerRowCellContent-isCurrent {
+    color: $localNavTextColor; /* 1 */
+    border: 1px dotted $globalSubduedTextColor;
```

Comment:
Don't like the dotted here. It's another boxy element (4 of them). I'd go with the color of the background hover state (blue) for the text, and leave the rest of the design elements off. Similar to the original design.

![image](https://cloud.githubusercontent.com/assets/324519/25353248/7255db58-28e3-11e7-9a98-c20f04bf0c1a.png)